specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json

```diff
@@ -482,6 +518,26 @@
         }
       }
     },
+    "RoomsCollection": {
+      "description": "A collection of existing rooms.",
```

Comment:
A collection of rooms. 

Just to stay consistent with the participant collection as well